app/src/main/java/com/android/androidfundamentalsgroup1/Activity_CC_2_2.java

```diff
@@ -0,0 +1,50 @@
+package com.android.androidfundamentalsgroup1;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.os.Bundle;
+import android.widget.TextView;
+
+public class Activity_CC_2_2 extends AppCompatActivity {
```

Comment:
For class names please avoid using _. Use camel case. For this case it looks like this: ActivityCC22